tests/unit/package_managers/test_gomod.py

```diff
@@ -2455,7 +2455,7 @@ class TestGoWork:
         ],
     )
     @mock.patch("cachi2.core.package_managers.gomod.run_cmd")
-    def test__init__(
+    def test_init__(
```

Comment:
`test_init__` - I would hope that the trailing underscores would be gone too :), looks weird otherwise.

Reply:
Done.